install-dev/data/xml/tab.xml

```diff
@@ -205,7 +205,7 @@
                     <class_name>AdminThemesCatalog</class_name>
                 </tab>
                 <tab id="Mail_Theme_Parent" id_parent="Look_feel" active="1" enabled="1" hide_host_mode="0" icon="">
-                  <class_name>AdminMailThemeParent</class_name>
+                  <class_name>AdminParentMailTheme</class_name>
```

Comment:
@atomiix Can you confirm this is expected ?

Reply:
Yes this is the expected value: https://github.com/PrestaShop/PrestaShop/commit/1f38d27d33916dbcc24ecc76d76894c5c42c9f10#diff-d2d10cd0f2a1915857d4e90f2386ac7a